test/fixtures/azure-fixture/src/main/java/fixture/azure/AzureHttpFixture.java

```diff
@@ -186,6 +251,9 @@ protected void after() {
         if (protocol != Protocol.NONE) {
             server.stop(0);
             metadataServer.stop(0);
+            if (oauthTokenServiceServer != null) {
```

Comment:
I think I'd be tempted just to always start this fixture, seems like it'll be simpler that way.